packages/grid/_modules_/grid/models/colDef/gridColDef.ts

```diff
@@ -71,6 +71,10 @@ export interface GridColDef {
    * @default 'string'
    */
   type?: GridColType;
+  /**
+   * Used in combination with `type='select'`. Array with possible cell values.
+   */
+  valueOptions?: Array<string>;
```

Comment:
We use more the foo[] syntax.

```suggestion
  valueOptions?: string[];
```

Reply:
It was a generic type initially